[Filebeat] backwards compatibility for set processor (#20908)
elastic/beats: CHANGELOG.next.asciidoc

- Fix long registry migration times. {pull}20717[20717] {issue}20705[20705]
- Fix event types and categories in auditd module to comply with ECS {pull}20652[20652]
- Update documentation in the azure module filebeat. {pull}20815[20815]

> - provide backwards compatibility for set processor and elasticsearch less than 7.9.0 {pull}20908[20908]

Reviewer: ```suggestion
- Provide backwards compatibility for the `set` processor when Elasticsearch is less than 7.9.0. {pull}20908[20908]
```
